### ISSUE TITLE
reduce sampling

### ISSUE THREAD

**Issue by QuerQue** [GoogleCloudPlatform/prometheus-engine#476]
Hi, not an issue but question about possible workarounds.
I have on-prem with Managed Service for Prometheus and node_exporter deployed.
One of metric I need is node_cpu_secods_total/counter. Everything works fine I can monitor this metric in GCP Monitoring. The problem is that this single metric produces milions of samples monthly. 
My setup: 2 servers, Thread(s) per core: 2, Core(s) per socket: 12 ,so 24 cpus, 8 modes (idle, iowait, irq).
Scraping interval 30s.
To sum up, 2x24x8 = 384 samples/30s ~= 13 samples/sec.
I know that I can change interval to e.g. 60s but still 90% of ingested samples comes from this single metric.
The question is, can I pull only idle mode? I don't need the rest of the modes.
I assume that I can't do anything with dividing 2 physical CPU to 24 (cores*threads).

Thanks for any advice.

**Comment by lyanco** [comment]
You should be able to do this with a `drop` or `keep` action in the `metricRelabeling` section. 

something like

metric_relabel_configs:
  - source_labels: [mode]
    regex: 'idle'
    action: keep

Although syntax may vary depending on how you configure Prometheus

**Comment by pintohutch** [comment]
Resolving per the advice provided in the previous comment. Please re-open if there are further questions.
